components/HeaderBlock.vue

```diff
@@ -22,15 +29,25 @@ export default {
 
   data () {
     return {
-      streamServer
+      streamServer,
+      timestamp: ''
     }
   },
   computed: {
 
   },
+  created () {
+    setInterval(this.getNow, 1000)
+  },
+
   mounted () {
   },
   methods: {
+    getNow () {
```

Comment:
Question for the future: 
the `getNow()` function might be global function called on different multiple components, should we move it to some other place? 

Perhaps we will add /schedule page where we display crosshair to show the user where we are in the program. That crosshair could use the exact same method (`getNow()`) and exact same logic (`setInterval(this.getNow, 1000)`)

Reply:
hm, good idea for the crosshair. also topic for Saturday?
`getNow` can be a mixin function, whynot?